homeassistant/helpers/typing.py

```diff
@@ -1,12 +1,39 @@
 """Typing Helpers for Home-Assistant."""
 from typing import Dict, Any
 
-import homeassistant.core
+# NOTE: NewType added to typing in 3.5.2 in June, 2016; Since 3.5.2 includes
+#       security fixes everyone on 3.5 should upgrade "soon"
+try:
+    from typing import NewType
+except ImportError:
+    NewType = None
+
+# HACK: mypy/pytype will import, other interpreters will not; this is to avoid
```

Comment:
I disagree with this hack. 

`homeassistant.core` should be independent and all `util` packages should not be aware of Home Assistant thus should be importable.